README.md

```diff
@@ -153,6 +154,26 @@ To configure annotation processing in IntelliJ:
 
 5. Install the lombok plugin in intellij.
 
+#### Further configuration in IntelliJ 
+
+* When working on the Pulsar core modules in IntelliJ, reduce the number of active projects in IntelliJ to speed up IDE actions and reduce unrelated IDE warnings.
+  * In IntelliJ's Maven UI's tree view under "Profiles"
+    * Activate "core-modules" maven profile
```

Comment:
use "Maven" instead of ""maven.Please check through the whole updates.

Reply:
done